project/AppDependencies.scala

```diff
@@ -0,0 +1,26 @@
+import sbt._
+import play.sbt.PlayImport._
+import play.core.PlayVersion
+import uk.gov.hmrc.SbtAutoBuildPlugin
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
+import uk.gov.hmrc.versioning.SbtGitVersioning
+
+object AppDependencies {
+
+  val compile = Seq(
+    ws,
+    "uk.gov.hmrc" %% "bootstrap-play-25" % "4.6.0",
+    "uk.gov.hmrc" %% "auth-client" % "2.4.0",
+    "uk.gov.hmrc" %% "domain" % "5.2.0"
+  )
+
+  val test = Seq(
+    "org.scalatest" %% "scalatest" % "2.2.6",
+    "org.pegdown" % "pegdown" % "1.6.0",
+    "org.mockito" % "mockito-core" % "1.9.0",
+    "com.typesafe.play" %% "play-test" % PlayVersion.current,
```

Comment:
Should this be set to 2.5.19 explicitly?

Reply:
Or whatever value is required?

Reply:
It's set automatically by whatever version of Play you're using. So by updating the play version in the plugins.sbt file this will be updated too - it saves having to set the play version twice in two different places (and removing the risk of making a mistake such that they end up both using different versions).

Reply:
Ok cool 👍